src/signals.c

```diff
@@ -633,8 +633,18 @@ void on_any_widget_icon_release_event(GtkWidget *widget,
  *                                                                     *
  ***********************************************************************/
 
+#if HAVE_VTE
+#if VTE_CHECK_VERSION(0,36,1)
```

Comment:
Is this the right check?

https://developer-old.gnome.org/vte/0.36/VteTerminal.html

```
void
user_function (VteTerminal *vteterminal,
               gpointer     user_data)
```

https://developer-old.gnome.org/vte/0.38/VteTerminal.html

```
void
user_function (VteTerminal *vteterminal,
               gint         status,
               gpointer     user_data)
```

Reply:
```
$ git tag --contains 08c778bf4a23abe614bad294027b4e88f6b727f0 | sort -n | head -n 5
0.37.0
```

Is this the right commit? https://gitlab.gnome.org/GNOME/vte/-/commit/08c778bf4a23abe614bad294027b4e88f6b727f0

Reply:
Yes, this is the right commit, `08c778bf4a23abe614bad294027b4e88f6b727f0`, and you're correct, the right check is for version 0.37.0 and not 0.36.1.   Thanks for double-checking it.